pysatModels/utils/extract.py

```diff
@@ -541,24 +565,43 @@ def instrument_view_irregular_model(inst, model, inst_name, mod_name,
 
     # Determine the scaling between model and instrument data
     inst_scale = np.ones(shape=len(inst_name), dtype=float)
-    for i, iname in enumerate(inst_name):
+    for i, (iname, mname) in enumerate(zip(inst_name, mod_name)):
```

Comment:
Would it not be clearer to use `mod_name[i]` instead of both enumerating and zipping?